core/modules/layout/layout.module

```diff
@@ -2768,3 +2768,17 @@ function _layout_clean_custom_css($classes = array()) {
 function layout_field_delete_instance($instance) {
   cache('layout_path')->flush();
 }
+
+/**
+ * Render a block
+ * @param Block $block
+ * The Block entity to render
```

Comment:
```suggestion
 * Render a block
 *
 * @param Block $block
 *   The Block object to render
```

Technically, the Block is not extending the Entity class so let's not call it that.